tests/test_login.py

```diff
@@ -1,10 +1,15 @@
 from fastapi.testclient import TestClient
 from app.main import app
+from tests import utils
 
 client = TestClient(app)
 
-
 def test_login():
+    utils.create_test_user()
     res = client.post("/login", data={"username": "Test", "password": "1234"})
+    utils.delete_test_user()
```

Comment:
Probably, it makes sense to delete the user at the end of the tests. 
Please investigate how fixtures work in tests and how to achieve automatic user creation/deletion using the fixtures instead set upping the test manually